### PR TITLE
util: make wordpress paginator also work with an array

### PR DESCRIPTION
**Pull request by cosimoc**
https://phabricator.endlessm.com/T21595

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -387,13 +387,24 @@ function _fetch_rss_page (feed, page, items, all_items, max_items, oldest_date)
 
 exports.fetch_rss_entries = fetch_rss_entries;
 
+function create_wordpress_paginator (feed) {
+    return function (n) {
```
Comment (sgnn7) [inline lib/util.js]: Might be easier to read with `n` expanded to a full var name

Comment (cosimoc) [reply lib/util.js]: Fixed

**Comment by sgnn7** [comment]
Should I merge this or leave it to @manuq to look over too?

**Comment by cosimoc** [comment]
The review request was for either of you :-)
Let's get it in!

**Pull request merged**